src/daft-plan/Cargo.toml

```diff
@@ -8,6 +21,7 @@ common-py-serde = {path = "../common/py-serde", default-features = false}
 common-treenode = {path = "../common/treenode", default-features = false}
 daft-core = {path = "../daft-core", default-features = false}
 daft-dsl = {path = "../daft-dsl", default-features = false}
+daft-functions = {path = "../daft-functions", default-features = false}
```

Comment:
we should only need this a dev dependency right? It looks like today we only use it in a test

Reply:
good call! will update.